build-parent/pom.xml

```diff
@@ -1058,17 +1058,5 @@
                 <script.extension>bat</script.extension>
             </properties>
         </profile>
-
-        <profile>
-            <id>jdk16-workarounds</id>
```

Comment:
This slipped through in my recent review: Please don't remove this just yet, there are still other, non-Gradle tests that are skipped on Java 16 (until fixed).